### PR TITLE
Create Subgraph project README

### PR DESCRIPTION
**Pull request by mtullyoc**
[Linear BAL-91](https://linear.app/bleu-llc/issue/BAL-91/create-a-readme-with-anvil-graph-node-setup-instructions)

### CODE REVIEW

File: packages/subgraph/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
+
+<h1 align="center"> Balancer Pool Metadata - Subgraph</h1>
```
Comment (ribeirojose) [inline packages/subgraph/README.md:3]:
```suggestion
<h1 align="center"> Balancer Pool Metadata - Subgraph</h1>
```

File: packages/subgraph/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# Getting started
+
+## Prerequisites
+
+Ensure you have installed all project dependencies and deployed the PoolMetadataRegistry contract to local testnet or another network, saving its contract address and deploy block.
+
+## Install Graph-node
+
+Should we address the graph-node here?
```
Comment (ribeirojose) [inline packages/subgraph/README.md:15]: is this a meta comment?

Comment (mtullyoc) [reply packages/subgraph/README.md]: Sorry... forgot the last push

File: packages/subgraph/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Ensure you have installed all project dependencies and deployed the PoolMetadataRegistry contract to local testnet or another network, saving its contract address and deploy block.
+
+## Install Graph-node
+
+```bash
+git clone https://github.com/graphprotocol/graph-node/
+cd graph-node/docker
+./setup.sh
+docker compose start
```
Comment (ribeirojose) [inline packages/subgraph/README.md:17]: probably
```suggestion
docker-compose start
```

File: packages/subgraph/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+
+## Setting up environment
+
+Define the networks with its respective address and startBlock on networks.yaml file:
+
+```yaml
+mainnet:
+  network: mainnet
+  EventEmitter:
```
Comment (ribeirojose) [inline packages/subgraph/README.md:28]:
```suggestion
  PoolMetadataRegistry:
```

**Review by ribeirojose — APPROVED** [review]
lgtm, would wait @mendesfabio 's review before merging

File: packages/subgraph/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  PoolMetadataRegistry:
+    address: "0x000000000000000000000000000000000000000"
+    startBlock: ??
+goerli:
+  network: goerli
+  PoolMetadataRegistry:
+    address: "0x000000000000000000000000000000000000000"
+    startBlock: ??
+
+```
```
Comment (mendesfabio) [inline packages/subgraph/README.md:37]:
```suggestion
    startBlock: ??
```

```

**Review by mendesfabio — APPROVED** [review]

**Pull request merged**